Source/Android/app/build.gradle

```diff
@@ -89,3 +94,14 @@ dependencies {
     // Allows FRP-style asynchronous operations in Android.
     compile 'io.reactivex:rxandroid:1.2.1'
 }
+
+def getVersion() {
+    String result
+    if(Os.isFamily(Os.FAMILY_WINDOWS)) {
+        result = 'cmd /c git describe --always --long'.execute([], project.rootDir).text
+    } else {
+        result = 'git describe --always --long'.execute([], project.rootDir).text
```

Comment:
Can you only have one call and put the "cmd /c" part in a prefix that's empty when you're not on Windows?